packages/storybook/stories/va-text-input-uswds.stories.jsx

```diff
@@ -58,6 +58,9 @@ const defaultArgs = {
   'form-heading': null,
   'form-description': null,
   'charcount': false,
+  'max': undefined,
+  'min': undefined,
+  'currency': undefined
```

Comment:
This is just me thinking out loud here but I'm wondering why in some situations we made these default props `null` but others `undefined`. Should we start making this consistent? I am guilty of doing this myself.